dockstore-cli-integration-testing/src/test/java/io/dockstore/client/cli/WDLWorkflowIT.java

```diff
@@ -16,13 +16,17 @@
 
 package io.dockstore.client.cli;
 
-import java.io.IOException;
```

Comment:
There's already a ticket on consistent imports sorting; I think these changes match the web service's layout, but ultimately that ticket should solve.